### PR TITLE
Shields for biostar, irc, travis in README.

### PR DESCRIPTION
**Pull request by jxtx**

### CODE REVIEW

**Comment by hexylena** [comment]
:+1:

**Comment by bgruening** [comment]
:+1: :-)

**Comment by dannon** [comment]
+1

File: README.rst [hunk 1]
```diff
@@ -3,6 +3,15 @@
 
 The latest information about Galaxy is available via `http://galaxyproject.org/ <http://galaxyproject.org/>`__
 
+.. image:: https://img.shields.io/badge/questions-galaxy%20biostar-blue.svg
+    :target: https://biostar.usegalaxy.org
+
+.. image:: https://img.shields.io/badge/chat-irc.freenode.net%23galaxyproject-blue.svg
+    :target: webchat.freenode.net/?channels=galaxyproject
```
Comment (martenson) [inline README.rst:10]: this is a dead link when following from github - GH thinks it is relative

Comment (hexylena) [inline README.rst:10]: #369

**Pull request merged**